julia/LibCEED.jl/src/LibCEED.jl

```diff
@@ -149,15 +149,14 @@ include("Request.jl")
 include("Operator.jl")
 include("Misc.jl")
 
-const minimum_libceed_version = v"0.8.0"
+const minimum_libceed_version = v"0.10.0"
```

Comment:
@jeremylt, do you prefer if this is 0.10.0 or 0.9.1?

Reply:
It looks like our next version number will be 0.10.0, so this should be good